crates/flox/doc/flox-run.md

```diff
@@ -11,21 +11,69 @@ flox-run - run app from current project
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] run [ `<options>` ] [ -- [ `<command args>` ... ] ]
+flox [ `<general-options>` ] run [ `<run-options>` ] [ `<installable>` ] [ -- [ `<command args>` ... ] ]
 
 # DESCRIPTION
 
-Run flake application from the requested package (or "installable").
-If not provided `flox` will prompt for you to select from the list of known packages.
+Run a flake application from the requested installable.
+See the [nix(1)] manual's section on installables for more information.
+
+[nix(1)]: https://nixos.org/manual/nix/stable/command-ref/new-cli/nix.html#installables
```

Comment:
We should definitely have some documentation about installables in our docs and link to to those as we do resolve a superset of nix installables.
Also the prompting is not mentioned anymore, as well as the fact that prompts only work in interactive usages, i.e. when a tty is available.

Reply:
Would it be sufficient to create a new issue in floxdocs to address this? But get these changes in as is?